bin/envconfig

```diff
@@ -72,15 +72,6 @@ if [ "$finish" = true ]; then
   return
 fi
 
-if [[ -n $BASH_VERSION ]]; then
```

Comment:
What's this change about? It's not part of the stated goal of this commit.

Reply:
Committed by mistake.